lib/publish-task.js

```diff
@@ -70,7 +70,7 @@ export async function publishTask(taskUrl, body, headers = {}, queue = "default"
         throw err;
       }
     },
-    { maxRetries }
+    { maxRetries: headers["x-no-retry"] ? 0 : maxRetries }
```

Comment:
Den här vill vi nog inte ändra på. Det här är retry-loopen mot Cloud Tasks, den kickar bara in om publiceringen mot Google failar. Borde kanske ha döpt den bättre...

Samma längre ner i den här filen.